src/wallet/rpcwallet.cpp

```diff
@@ -2572,6 +2572,37 @@ static RPCHelpMan listwallets()
     };
 }
 
+static std::tuple<std::shared_ptr<CWallet>, std::vector<bilingual_str>> LoadWallet(WalletContext& context, UniValue load_on_start_param, const std::string wallet_name)
```

Comment:
In 5b0a506962fe4388db13062e105201773954e4ee "Add a new RPC command: restorewallet"

nit: I would prefer for this to not be named the same as the `LoadWallet` function in wallet.cpp

Reply:
Done. Changed the name to `LoadWalletHelper`.